staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/mutating_work_estimator.go

```diff
@@ -129,3 +138,12 @@ func (e *mutatingWorkEstimator) estimate(r *http.Request, flowSchemaName, priori
 		AdditionalLatency: additionalLatency,
 	}
 }
+
+func isRequestExemptFromEvents(requestInfo *apirequest.RequestInfo) bool {
+	// Creating token for service account does not produce any event,
+	// but still serviceaccounts can have multiple watchers.
+	if requestInfo.Resource == "serviceaccounts" && requestInfo.Subresource == "token" {
```

Comment:
so "/api/v1/namespaces/kube-system/serviceaccounts/default/token"  does not change the underlying service account object, and clients watching service accounts in this namespace would receive no watch event, right?
so, are there other requests with sub resource that does not produce watch events?

Reply:
> so "/api/v1/namespaces/kube-system/serviceaccounts/default/token" does not change the underlying service account object, and clients watching service accounts in this namespace would receive no watch event, right?

correct

> so, are there other requests with sub resource that does not produce watch events?
I wasn't able to find any other (there are still some long running subresources like proxy/logs etc, but they are exempt from P&F). Similarly there is also tokenreviews api, but you can only create tokens and not watch this resource.